app/controllers/agenda/agendaitems/agendaitem/index.js

```diff
@@ -36,14 +36,15 @@ export default class IndexAgendaitemAgendaitemsAgendaController extends Controll
     const neighbouringItem = await this.store.queryOne('agendaitem', {
       'filter[agenda][:id:]': this.agenda.id,
       'filter[type][:id:]': agendaItemType.id,
-      'filter[:lte:number]': `"${previousNumber}"`, // Needs quotes because of bug in mu-cl-resources
+      'filter[number]': previousNumber, // Needs quotes because of bug in mu-cl-resources
```

Comment:
Is the bug is fixed? (we do run a new version of mu-cl-resource).
Been so long I don't even know how to properly test.
But then the comments can be removed aswell